scripts/run_mypy_checks.py

```diff
@@ -769,37 +774,65 @@ def install_third_party_libraries(skip_install):
         install_third_party_libs.main()
 
 
-def get_mypy_cmd(files):
+def get_mypy_cmd(files, ci_check):
     """Return the appropriate command to be run.
 
     Args:
         files: list(list(str)). List having first element as list of string.
+        ci_check: bool. Whether the test is for CI.
```

Comment:
I think DubeySandeep's comment on the argument above applies here too. It would be clearer to have the argument specify whether mypy should be installed globally than to specify whether we are running on CI

Reply:
Done